hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/transfer/ClassicTransfersCall.java

```diff
@@ -74,7 +76,7 @@ public ClassicTransfersCall(
             @Nullable ApprovalSwitchHelper approvalSwitchHelper,
             @NonNull final VerificationStrategy verificationStrategy,
             @NonNull final SystemAccountCreditScreen systemAccountCreditScreen) {
-        super(enhancement);
+        super(gasCalculator, enhancement);
```

Comment:
Commont `// TODO - gas calculation` at L95 and thus there are several `0L` gas charges in there.

Reply:
Fixed.